src/components/@shared/AssetList/index.tsx

```diff
@@ -52,14 +52,15 @@ export default function AssetList({
   useEffect(() => {
     if (!assets) return
     setAssetsWithPrices(assets as AssetExtended[])
-    setLoading(false)
+    setLoading(true)
     async function fetchPrices() {
       const assetsWithPrices = await getAccessDetailsForAssets(
         assets,
         accountId || ''
       )
       if (!isMounted()) return
```

Comment:
If it returns here, it will be permanently loading

Reply:
We can actually remove this line as it doesn't really affects anything, but I'm not sure what you're referring to. How do we replicate this case?

Reply:
Yeah, it's not something easy to replicate. I was just thinking about the implications of setLoading being set to true before this while previously it was false, potentially that could mean people see the loading spinner for longer in some situations

Reply:
This line's goal is to prevent setting state on a component which is not rendered on screen anymore, leading to memory leaks. This can happen when users load front page and while app is fetching data, user browses to another route, unmounting the component. Without this line, React will then try to update state on the unmounted component